src/connector.ts

```diff
@@ -134,4 +142,22 @@ export class Connector {
 
 		addedChanged(unprotectString(this.coreHandler.core.deviceId))
 	}
+
+	private setupReloadDataKoaEndpoint(): void {
+		const KOA_PORT: number = 3007
+		const RUNDOWN_EXTERNAL_ID_SUFFIX: string = '_1'
+
+		this.koaRouter.get('/reloadData/:rundownName', async (ctx, next): Promise<void> => {
```

Comment:
Perhaps another method than GET should be used, since we are not reading the data out, but triggering a computation.

Reply:
I did think about this, what is more correct in this scenario? POST and PUT, for instance, would signify that the we actually submit something rather than trigger a computation, right?

Reply:
For now it has been changed to a POST.